src/main/groovy/io/saagie/plugin/dataops/clients/SaagieClient.groovy

```diff
@@ -951,6 +956,81 @@ class SaagieClient {
         }
     }
 
+    String exportJob() {
+        logger.info('Starting Export Job task')
+
+        checkRequiredConfig(!configuration?.project?.id ||
+                            !configuration?.job?.id ||
+                            !configuration?.export?.export_file_path ||
+                            !configuration?.export?.overwrite
+        )
+        def result = getJobAndJobVersionDetailToExport();
+        logger.debug(result);
+        File newDirectoryStructureParent = new File(configuration.export.export_file_path);
+
+    }
+
+    String getJobAndJobVersionDetailToExport() {
+
+        def jsonGenerator = new JsonGenerator.Options()
+            .excludeNulls()
+            .build()
+
+        checkRequiredConfig(!configuration?.project?.id ||
+            !configuration?.job?.id ||
+            !configuration?.export?.export_file_path ||
+            !configuration?.export?.overwrite
+        )
+
+        Request getJobDetail = saagieUtils.getJobDetailRequest()
+        def job = configuration.job;
+        ExportJob exportJob = null;
+        def projectId = configuration.project.id;
+        try {
+            client.newCall(getJobDetail).execute().withCloseable { response ->
+                handleErrors(response)
+                String responseBody = response.body().string()
+                def parsedResult = slurper.parseText(responseBody)
+                if (parsedResult.data == null) {
+                    def message = "Something went wrong when getting job detail: $responseBody for job id $jobId and $projectId"
+                    logger.error(message)
+                    throw new GradleException(message)
+                } else {
+                    Job jobDetailResult = parsedResult.data.job
+                    jobDetail = JsonOutput.toJson(jobDetailResult)
+
+                    exportJob.job = jobDetailResult
+                    if(!jobDetailResult.versions.isEmpty()) {
+                        jobDetailResult.versions.sort { it.creationDate }
+                    }
+                    jobDetailResult.versions.each {
+                        if(it.isCurrent) {
+                            exportJob.jobVersion = it
+                        }
+                        if(it.packageInfo && it.packageInfo.downloadUrl) {
+                            exportJob.downloadUrl =  it.packageInfo.downloadUrl
+                        }
+                    }
+
+                    if(!exportJob.downloadUrl) {
+                        def message = "The is no download URl"
```

Comment:
"There is no download URL" => il faut voir si il est nécessaire d'avoir un package avec un fichier pour créer l'export, ou si seule la config du job suffit